docs/.gdnbaselines

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "latest",
+  "baselines": {
+    "default": {
+      "name": "default",
+      "createdDate": "2022-03-03 04:59:14Z",
+      "lastUpdatedDate": "2022-03-03 04:59:14Z"
+    }
+  },
+  "results": {
+    "552baaa9d24596ed82e97b538dbef8abfcc9a0a8a9fdc8d3423424bf003b4fa7": {
+      "signature": "552baaa9d24596ed82e97b538dbef8abfcc9a0a8a9fdc8d3423424bf003b4fa7",
+      "alternativeSignatures": [
+        "b5f214e29b5462952a2841914ef8d1fc8141a9e8a69ea2b2be6c0cd409bf15d7"
+      ],
+      "target": "docs/themes/thxvscode/assets/js/bundle.js",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "eslint",
+      "ruleId": "@microsoft/sdl/no-inner-html",
+      "justification": "Comes from jquery dependency. Tracked by https://github.com/microsoft/FluidFramework/issues/9316.",
```

Comment:
This file defines the "baseline errors" that we have right now and tracks that they get resolved by the expiration date.